pkg/config/config.toml.nextgen.example

```diff
@@ -0,0 +1,480 @@
+# TiDB Configuration.
+
+# TiDB server host.
+host = "0.0.0.0"
+
+# tidb server advertise IP.
+advertise-address = ""
+
+# TiDB server port.
+port = 4000
+
+# Registered store name, [tikv, mocktikv, unistore]
+store = "unistore"
+
+# TiDB storage path.
+path = "/tmp/tidb"
+
+# The socket file to use for connection.
+socket = "/tmp/tidb-{Port}.sock"
+
+# The Keyspace Name of the tidb
+keyspace-name = "default"
```

Comment:
```suggestion
keyspace-name = "SYSTEM"
```
as `SYSTEM` must be bootstrapped before any other user keyspace, and itself can also be used as a user keyspace, i prefer we use this one